sample-dcrstakepool.conf

```diff
@@ -55,3 +55,6 @@ testnet=1
 
 ; Useful for debugging RPC client issues.
 ;debuglevel=trace
+
+; Admin IPs to limit where certain pages can be accessed for admin maintainance (Status et al)
+;adminips=127.0.0.1
```

Comment:
I would uncomment 127.0.0.1 since that's safe and it will bomb if it's not specified.  Also wrap at 80 lines. Maybe something like:

```
; Admin-specific functionality like the status page are limited by an IP address filter.
adminips=127.0.0.1
; Multiple values can be used and are separated by a comma.
;adminips=127.0.0.1,192.0.2.1,198.51.100.1
```

Also need to add minservers.